### PR TITLE
Feature/pact poc

### PR DESCRIPTION
**Pull request by kevinshahfws**

### CODE REVIEW

File: device/thunder/src/lib.rs [hunk 1]
```diff
@@ -34,3 +34,8 @@ pub mod processors {
     pub mod thunder_wifi;
     pub mod thunder_window_manager;
 }
+
+pub mod tests {
```
Comment (satlead) [inline device/thunder/src/lib.rs:38]: Lets modularize this a bit further 
create a folder for contracts
-- contracts
-- device_info_ptest.rs

Comment (kevinshahfws) [reply device/thunder/src/lib.rs:38]: device_info means only test case related to device.info method or all the device thunder tests??

Comment (kevinshahfws) [reply device/thunder/src/lib.rs:38]: done

**Review by satlead — APPROVED** [review]
We probably need to build this once before merging. So please hold off on the merge

File: device/thunder/Cargo.toml [hunk 2]
```diff
@@ -29,3 +29,14 @@ crate-type = ["cdylib"]
...
 thunder_ripple_sdk = { path = "../thunder_ripple_sdk" }
 serde = { version = "1.0", features = ["derive"] }
+pact_consumer = {version = "=0.10.4", optional = true}
+reqwest = {version = "0.11", optional = true}
+expectest = {version = "0.12.0", optional = true}
+pact-plugin-driver = {version = "0.4.1", optional = true}
+pact_models = {version = "1.0.9", optional = true}
+maplit = {version = "1.0.2", optional = true}
+test-log = {version = "0.2.11", optional = true}
+url = {version = "2.2.2", optional = true}
```
Comment (adamdama) [inline device/thunder/Cargo.toml:39]: @kevinshahfws are these not better suited to the [dev-dependencies] section of the toml as they are only used in testing and not in prod?

Comment (kevinshahfws) [reply device/thunder/Cargo.toml:39]: Will close this PR... Now changes are in PR #170 
But to answer, yes it is purely for contract_tests feature specific

**Comment by satlead** [comment]
#170 Replaces this

**Pull request closed without merging**